src/components/KRadio/KRadio.vue

```diff
@@ -101,34 +119,37 @@ const props = defineProps({
     type: [String, Number, Boolean, Object],
     required: true,
   },
+  hasError: {
+    type: Boolean,
+    default: false,
+  },
+  isCard: {
+    type: Boolean,
+    default: false,
+  },
   /**
-   * Controls appearance of radio input element
+   * @deprecated in favor of `isCard`
    */
   type: {
     type: String as PropType<RadioTypes>,
     default: 'radio',
-    validator: (value: RadioTypes): boolean => RadioTypesArray.includes(value),
-  },
-  /**
-   * Test mode - for testing only, strips out generated ids
-   */
-  testMode: {
-    type: Boolean,
-    default: false,
+    validator: (value: RadioTypes): boolean => {
+      if (value === 'card') {
```

Comment:
question: shouldn't we warn if this prop is used at all? Not just for `card`

Reply:
So then should the default prop value be removed? Then the display is fine (since we don't use this prop) and if they provide a value for it, you can console.warn

Reply:
Ah fair enough, updated this